go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/bronze1man/yaml2json v0.0.0-20211227013850-8972abeaea25
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/gardener/dependency-watchdog v0.7.0
-	github.com/gardener/etcd-druid v0.15.0
+	github.com/gardener/etcd-druid v0.15.3
```

Comment:
https://github.com/gardener/gardener/pull/7248 Already tackles this. And will be merged soon.